### PR TITLE
Adding gitpod configuration

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

**Review by billyoung — DISMISSED** [review]
:+1:

**Comment by autodidaddict** [comment]
This doesn't quite work yet... or at least, I'm still waiting to see the prebuild happen on gitpod.

**Comment by autodidaddict** [comment]
To try this out, open the following link: [here](https://gitpod.io/#https://github.com/wasmCloud/wasmCloud/pull/162)

**Comment by brooksmtownsend** [comment]
@autodidaddict I've been in the code weeds for a little bit, getting to it now

File: .gitpod.yml
Comment (brooksmtownsend) [inline .gitpod.yml:9]: We may want to wait to merge the gitpod config until `0.4.0` is released for `wash-cli`. It also brings up a valid point that the latest released version of `wash` should ALWAYS match the latest release of `wasmcloud-host` + `wasmcloud-control-interface`

Comment (brooksmtownsend) [reply .gitpod.yml:9]: `wash-cli` `0.4.0` will be released once my no-nats PR is merged

**Review by brooksmtownsend — APPROVED** [review]
Non-blocking thought that we may want to hold off on this because the version of `wash` is too old to download provider-archive files (with the whole OCI artifact type). Once https://github.com/wasmCloud/wasmCloud/pull/163 is merged, https://github.com/wasmCloud/wash/pull/107 will be ready to go, and `0.4.0` will be released. 

Regarding this PR, this Gitpod thing is _awesome_. We may want to introduce a little tutorial on how to get the rust analyzer / rustlang code suggestions up-and-running but in general I can't believe how easy it is to just jump into a full development environment with everything installed. With the ability to add remotes (for forking) and make a PR right there it's 🔥

File: .gitpod.yml
Comment (autodidaddict) [reply .gitpod.yml:9]: We can either merge this now and just update the files once wash is released or we can wait.. I can be convinced either way.

Comment (brooksmtownsend) [reply .gitpod.yml:9]: I think we should install from the latest release regardless, but I think we should wait so nobody hops in and tries to develop in an environment that isn't fully working yet. Apologies on letting `wash` get so far out of sync

**Pull request merged**
